modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -32,43 +32,32 @@
       it 'matches the expected schema' do
         # TODO: this should use the matcher helper instead (was throwing an Oj::ParseError)
         # expect().to match_json_schema('immunizations')
```

Comment:
Created a separate ticket to solve this issue. https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7614

Reply:
I think it may be the other way around if I'm reading that ticket right. Even if we combine the spec schemas and the doc schemas, the specs will still need to call a file to validate, in which case, when that stub is happening with the RSA key, it won't work. So 7614 will need to be solved before all specs can validate a schema.

Reply:
Sorry, I didn't really read the ticket. I was just looking at the code and assumed you meant fix the schema. Didn't realize it was caused by an auth stubbing issue.